src/test/java/LoginPage/LoginPage.java

```diff
@@ -0,0 +1,42 @@
+package LoginPage;
+
+import ScansPage.ScansPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.time.Duration;
+
+
+public class LoginPage {
+
+    private final WebDriver driver;
+    public LoginPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void open() {
+        driver.get("https://app.neuralegion.com/login");
+        driver.manage().timeouts().implicitlyWait(Duration.ofMillis(3000));
+    }
+
+    private final By emailField = By.id("mat-input-0");
```

Comment:
Please, move `private` properties after `public`. The order inside class should be:

1. Public properties
2. Protected properties
3. Private properties
4. Constructor
5. Public methods
6. Protected methods
7. Private methods